CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### 4.0.2 (2021-07-21)
+- Improvements:
+	- Changed campaigns to not display when attached image cannot be downloaded. [SDKCF-3977]
+	- Changed campaigns to not display upon display permission request failure. [SDKCF-3976]
+	- Increased clickable area of the IAM close button "x". [SDKCF-3958]
```

Comment:
Just one more recent fix: SDKCF-4020
https://github.com/rakutentech/ios-inappmessaging/pull/102

Reply:
Oops sorry, didn't mean to leave out your work. Kept thinking that ticket was a SDKCF-3958 regression

Reply:
No worries!